### DISCUSSION TITLE
Soil Moisture & Temperature & EC Sensor | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#967] [Q&A]
# Soil Moisture & Temperature & EC Sensor | Seeed Studio Wiki

Soil Moisture & Temperature & EC Sensor

https://wiki.seeedstudio.com/Sensor/SenseCAP/SenseCAP_Probe/Soil-Moisture-Temperature-EC-Sensor/

**Comment by jnkws** [comment]
Is there a schematic for the sensor?
At what frequency are the values measured?
